content/fa/docs/contribute/localization.md

```diff
@@ -0,0 +1,375 @@
+---
+title: بومی‌سازی مستندات کوبرنتیز
+content_type: concept
+approvers:
+- moh0ps
```

Comment:
Commenting Out Reviewer Information

Sometimes, the original code contains a reviewer/approvers specified at the top. Since the reviewer of the original page usually cannot review the localized page, the reviewer-related code in the resource metadata should be commented out. Also, that section in the head is not intended for localization approvers. :)

Below is an example of commenting out reviewer-related code:

```diff
- reviews:
- - reviewer1
- - reviewer
+ # reviews:
+ # - reviewer1
+ # - reviewer
```